checks/evaluation/cii_best_practices.go

```diff
@@ -32,7 +32,7 @@ const (
 )
 
 // CIIBestPractices applies the score policy for the CIIBestPractices check.
-func CIIBestPractices(name string, dl checker.DetailLogger, r *checker.CIIBestPracticesData) checker.CheckResult {
+func CIIBestPractices(name string, _ checker.DetailLogger, r *checker.CIIBestPracticesData) checker.CheckResult {
```

Comment:
Rather than the underscore, should we be using the logger at all?
If yes, might be worth it's own PR.

Reply:
Agreed. Another PR :)